tools/its.py

```diff
@@ -275,52 +298,39 @@ def driver_enter_code(self, driver, plz_impfzentrum):
             except:
                 pass
 
-    def driver_renew_cookies(self, driver, plz_impfzentrum):
```

Comment:
In die for Schleife muss noch ein `action = ActionChains(driver)`. Beim Buchen wird der richtige Code eingegeben und das DOM ändert sich dann. Sonst habe ich das Termin Buchen erfolgreich getestet. Die manuelle Selenium Buchung aber noch nicht.

Reply:
Ich hab hier ja nichts geändert. Wie kommt es, dass das jetzt nötig ist?

Reply:
Seither wurde immer ein falsche Code eingegeben. Dann ist die Seite gleichgeblieben und man wurde nicht weitergeleitet. So wie du es hast ist es aber besser.

Reply:
36a063bd2d2405dd51c255efe0f7b22045f9167c Passt das so?